mmdet3d/core/bbox/box_np_ops.py

```diff
@@ -38,11 +39,14 @@ def box_camera_to_lidar(data, r_rect, velo2cam):
     Returns:
         np.ndarray, shape=[N, 3]: Boxes in lidar coordinate.
     """
+    # TODO raw coords?
     xyz = data[:, 0:3]
-    l, h, w = data[:, 3:4], data[:, 4:5], data[:, 5:6]
+    dx, dy, dz = data[:, 3:4], data[:, 4:5], data[:, 5:6]
     r = data[:, 6:7]
     xyz_lidar = camera_to_lidar(xyz, r_rect, velo2cam)
-    return np.concatenate([xyz_lidar, w, l, h, r], axis=1)
+    r_new = -r - np.pi / 2
```

Comment:
need to leave a comment, e.g., link to tutorial

Reply:
Added a comment, link to tutorial can be added in the tutorial pr